masonite/request.py

```diff
@@ -36,12 +39,33 @@ def __init__(self, environ=None):
 
         self.encryption_key = False
         self.container = None
+        # Generate csrf token
+        self.generate_csrf_token()
+
+    def generate_csrf_token(self):
```

Comment:
Ok so let's move all these csrf methods out of the request class and into a new `Csrf` class.